dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/SerializationInfoCache.java

```diff
@@ -128,10 +129,12 @@ public byte getIndex() {
             return Long.class;
         }
 
+        @Trivial // reveals customer data
```

Comment:
Can mark these whole classes @Trivial if the serialize debug includes the type

Reply:
Yep, I'll update the whole class as trivial and trace the information in serialize/deserialize